src/coprocessor/codec/chunk/chunk.rs

```diff
@@ -75,8 +77,30 @@ impl Chunk {
     pub fn iter(&self) -> RowIterator {
         RowIterator::new(self)
     }
+
+    #[cfg(test)]
+    pub fn decode(buf: &mut BytesSlice, tps: &[FieldType]) -> Result<Chunk> {
```

Comment:
It's ported from tidb and now only the `u8` is used. I think we use the `FieldType`  just in case one day we may need some other attributes in `FieldType` @zz-jason  do you think so?

Reply:
+1, Agree with you.